src/arrayMethodSort.js

```diff
@@ -4,8 +4,24 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    let callbackCompare = compareFunction;
+
+    if (!callbackCompare) {
+      callbackCompare = (a, b) => {
+        return a.toString() > b.toString();
+      };
+    }
```

Comment:
```suggestion
  [].__proto__.sort2 = function (compareFunction = (a, b) => a.toString() > b.toString()) {
```
just add the default value